### PR TITLE
[NTOSKRNL] Sync some INIT_FUNCTION

### PR DESCRIPTION
**Pull request by SergeGautherie**
Addendum to 85e5b5be5688d9ea1b91c9bc4441beab72b71c14 (r49445).

Fix MSVC 2015 x86 custom build:
"...\ntoskrnl\kdbg\kdb.c(1699): error C2983: 'KdbpGetCommandLineSettings': all declarations must have an identical __declspec(code_seg(...))"

Please, double-check.

NB:
I did not try to sync arm/amd64/powerpc cases.

### CODE REVIEW

File: ntoskrnl/include/internal/cm.h [hunk 1]
```diff
@@ -815,6 +815,7 @@ CmpDestroyHive(
     IN PCMHIVE CmHive
 );
 
+//INIT_FUNCTION
```
Comment (ThFabba) [inline ntoskrnl/include/internal/cm.h:818]: This does not seem useful.

File: ntoskrnl/kdbg/kdb.c [hunk 2]
```diff
@@ -1691,7 +1691,7 @@ KdbEnterDebuggerException(
     return ContinueType;
 }
 
-INIT_FUNCTION
+//INIT_FUNCTION
```
Comment (ThFabba) [inline ntoskrnl/kdbg/kdb.c:1694]: Just remove please.

File: ntoskrnl/kdbg/kdb.h [hunk 3]
```diff
@@ -226,6 +226,7 @@ BOOLEAN
 KdbpAttachToProcess(
    PVOID ProcessId);
 
+//INIT_FUNCTION
```
Comment (ThFabba) [inline ntoskrnl/kdbg/kdb.h:229]: Just remove please.

File: ntoskrnl/include/internal/cm.h (hunk 1 above)
Comment (SergeGautherie) [reply ntoskrnl/include/internal/cm.h]: Removed comment in .c, instead.

File: ntoskrnl/kdbg/kdb.c (hunk 2 above)
Comment (SergeGautherie) [reply ntoskrnl/kdbg/kdb.c]: Done.

File: ntoskrnl/kdbg/kdb.h (hunk 3 above)
Comment (SergeGautherie) [reply ntoskrnl/kdbg/kdb.h]: Reverted.

**Pull request merged**